packages/cli/src/utils/deploy.ts

```diff
@@ -397,13 +399,26 @@ export async function deploy(
     try {
       const factory = new ethers.ContractFactory(abi, bytecode, signer);
       console.log(chalk.gray(`executing deployment of ${contractName} with nonce ${nonce}`));
-      const deployPromise = factory
-        .deploy({
-          nonce: nonce++,
-          maxPriorityFeePerGas,
-          maxFeePerGas,
-        })
-        .then((c) => (disableTxWait ? c : c.deployed()));
+      let deployPromise;
+
+      if (maxFeePerGas && maxPriorityFeePerGas) {
```

Comment:
I think we might need an explicit `== null` check or similar here because we set anvil to use `0` for `maxFeePerGas`?

Reply:
Thanks for the feedback. 
For anvil is that the default setting in order to make it EIP-1559 compliant? If yes - then I think it should be fine as is but I will test.